controllers/configurations.js

```diff
@@ -0,0 +1,18 @@
+const { allProducts } = require('./productsController');
+const attributes = require('../models').attributes;
+const attribures_values = require('../models').attribute_values;
```

Comment:
I suggest destructuring like to avoid reduncency.
const { attributes, attribures_values } = require('../models');